src/interactive/AccordionPanel.jsx

```diff
@@ -183,8 +199,8 @@ class AccordionPanel extends React.Component {
 				<div
 					role="tab"
 					aria-controls={`panel-${panelId}`}
-					aria-expanded={isOpen}
-					aria-selected={isOpen}
+					aria-expanded={isOpen && !isDisabledPanelOpen}
```

Comment:
no sure if this is correct
Panel should be considered opened if it has disabled content